src/main/webapp/main.js

```diff
@@ -0,0 +1,25 @@
+var imageList = [];
+
+function add(img) {
+	imageList.push(img);
+	var table = document.getElementById('selectionTable');
+	var newRow = table.insertRow();
+	
+	var textCell = newRow.insertCell();
+	textCell.appendChild(document.createTextNode(imageList[imageList.length - 1]));
+	
+	var buttonCell = newRow.insertCell();
+	var removeButton = document.createElement('button');
+	removeButton.type = 'button';
+	removeButton.innerHTML = 'Remove';
+	removeButton.onclick = remove; //does not show up in HTML but works
+	buttonCell.appendChild(removeButton);
+}
+
+var remove = function() {
+	document.getElementById('test').innerHTML = 'text';
```

Comment:
It doesn't look like you fully implemented this method yet.  Presumably when the user clicks on "Remove" the row should disappear and the ID should be removed from imageList, right?